spec/models/contact_type_group_spec.rb

```diff
@@ -3,8 +3,9 @@
 
 RSpec.describe ContactTypeGroup, type: :model do
   it "should not have duplicate names" do
-    name = create(:contact_type_group, {name: "Test1"})
-
-    expect { create(:contact_type_group, {name: "Test1"})}.to raise_error(ActiveRecord::RecordInvalid, "Validation failed: Name has already been taken")
+    org_id = create(:casa_org).id
+    create_contact_type_group = -> { create(:contact_type_group, {name: "Test1", casa_org_id: org_id }) }
+    create_contact_type_group.()
```

Comment:
fancy :)

Reply:
Thank you. ;) The reasons I like using the lambda are 1) it communicates more clearly that the 2 operations are identical, and 2) it separates low level behavior from higher level behavior, and names that behavior. I'm a big fan of this.